Curatour/frontend
# Feature/apollo integration

## *Purpose:*
*What does this merge do? Check all that apply.*


## *Description: What does this PR do?*
Implement apollo to query event data from API.
Store queried data from API in global state using context.
Utilize `useQuery` in `App.js` to query event data when application is opened.
Format event data so calendar package to interpret correctly.

## *Instructions for reviewer:*
*Where should the reviewer start?*
Lines 11-24 of `src/components/App.js`
Lines 9-11, 24-28 of `src/components/calendar/Calendar.js`
Lines 29-39 of `src/components/calendar/event-utils.js`
Lines 14 and 17 of `src/context/AppContext.js` can probably be cleaned up.
`src/context/queries.js`

*How should this be manually tested?*
Run the application on your local machine.
Events that were previously stored on the BE should appear.
For extra confirmation, use PostMan to add more events, using the `mutation` documentation from the BE server, THEN confirm the added event is displayed.

*Any background context you want to provide?*
![Screen Shot 2021-04-17 at 8 25 50 AM](https://user-images.githubusercontent.com/64617435/115116626-c91ca180-9f57-11eb-8404-ba1661d1f72e.png)
![Screen Shot 2021-04-17 at 8 16 47 AM](https://user-images.githubusercontent.com/64617435/115116628-cb7efb80-9f57-11eb-9458-6e4e2bbfbe5c.png)

*What are the relevant tickets?*

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*



## Checklist:
